samples/activity_trigger_types/ReturnFloat/__init__.py

```diff
@@ -0,0 +1,9 @@
+import json
+import logging
+
+logger = logging.getLogger('activity')
+
+
+def main(value: str):
```

Comment:
can we add a return type annotation here? (and in all other activity functions)

Reply:
There are two types of sample, one is noted with "with_anno" those activity triggers contain return type annotation, while others don't.